src/modules/AutoinstSoftware.rb

```diff
@@ -921,13 +885,21 @@ def Write
 
         Pkg.DoRemove(PackageAI.toremove)
       end
-      pack = Storage.AddPackageList
-      if Ops.greater_than(Builtins.size(pack), 0)
-        Builtins.y2milestone(
-          "Installing storage packages: %1",
-          Pkg.DoProvide(pack)
-        )
+
+      storage_pack = Storage.AddPackageList
+      log.info "Storage packages for installation: #{storage_pack}"
+      failed_packages = failed_packages.merge(Pkg.DoProvide(storage_pack)) unless storage_pack.empty?
+
+      unless failed_packages.empty?
+        log.error "Cannot select: #{failed_packages}"
+        not_selected = ""
+        failed_packages.each do |name,reason|
+          not_selected << "#{name}: #{reason}\n"
+        end
+        # TRANSLATORS: Warning text during the installation. %s is a list of package
+        Report.Error(_("These packages cannot be found in the software repositories:\n%s") % not_selected)
```

Comment:
What happens if the list of packages is long? Maybe that's something to improve in the future.